crates/debug/src/transform/attr.rs

```diff
@@ -252,6 +271,18 @@ where
                 pending_di_refs.insert(current_scope_id, attr.name(), offset);
                 continue;
             }
+            AttributeValue::DebugAddrBase(offset) => {
+                if offset.0 != unit.addr_base.0 {
+                    return Err(TransformError("unexpected DebugAddrBase value").into());
+                }
+                continue;
+            }
+            AttributeValue::DebugStrOffsetsBase(offset) => {
```

Comment:
Let's just silently ignore `DebugAddrBase` or `DebugStrOffsetsBase` on DIEs regardless their values. Standard is saying we shall not expect them there. Other tools will probably will do the same.